engine/proto/master.proto

```diff
@@ -180,7 +180,7 @@ message ScheduleTaskResponse {
 }
 
 message ExecWorkload {
-    int32 tp = 1;
+    int32 tp = 1; // 需要用 enum 么
     int32 usage = 2;
```

Comment:
We may need to define exactly what it means, cpu, memory, or storage.

Reply:
Yep, the workload here needs to be redesigned. It is useless currently.